src/views/template/customerorder.html

```diff
@@ -20,11 +20,11 @@
                         {{ quantity }}
                     </td>
                 </tr>
-                {{ /products }}
+                {{ /order.products }}
             </tbody>
         </table>
     </div>
     <div class="panel-footer">
-        Earn {{ orderCost }}
+        Earn {{ order.orderCost() }}
```

Comment:
Remove the parentheses perhaps? I got this working in the container template for `_usedCapacity`.